.ci/install_qemu.sh

```diff
@@ -91,9 +92,6 @@ build_and_install_qemu() {
 
 	echo "Install Qemu"
 	sudo -E make install
-
-	# Add link from /usr/local/bin to /usr/bin
-	sudo ln -sf $(command -v qemu-system-${QEMU_ARCH}) "/usr/bin/qemu-lite-system-${QEMU_ARCH}"
```

Comment:
OOI then, where will we be installing this qemu - will it be in /usr/local/bin still? Just wondering if there is now more 'danger' that somebody running the scripts locally might over-write their system installed qemu.

Reply:
How about checking if [`KATA_DEV_MODE`](https://github.com/kata-containers/tests#developer-mode) is set. If it is and if `/usr/local/bin/qemu-system-${QEMU_ARCH}` already exists (I'm assuming that's where it will install to?), this script will exit with an error?

Reply:
@jodh-intel  @grahamwhaley 

how about call it `kata-qemu-system-x86_64` and install  it in `/usr/bin` ?

Reply:
`kata-*` because it will have some kata specific patches (vmtemplating, 9p, virtio-fs, etc)

Reply:
That works for me, particularly for the CI scripts etc. We should probably check if this has any knock on effects to packaging? /cc @egernst @marcov @jcvenegas

Reply:
@devimc - good idea. But as @grahamwhaley says, let's get more feedback...

Reply:
If this is tests specific, installing as `/usr/bin/qemu-system...` should be fine, making sure to check `KATA_DEV_MODE` as proposed by @jodh-intel and to not overwrite an existing binary anyway.

When/if packaging a patched QEMU, then we'll need to figure out the correct name / location.

Reply:
@jodh-intel @marcov I agree, if `KATA_DEV_MODE` is set and `qemu-system-$QEMU_ARCH` is installed, then exit with an error, otherwise configure, build and install qemu. By default it's installed in `/usr/bin`